src/components/App/App.jsx

```diff
@@ -1,14 +1,34 @@
+/* eslint-disable react/destructuring-assignment */
 import './App.css';
+import { Component } from 'react';
 import Display from '../Display/display.component';
 import ButtonPanel from '../ButtonPanel/button-panel.component';
-// eslint-disable-next-line no-unused-vars
 import calculate from '../logic/calculate';
 
-const App = () => (
-  <div className="main-container">
-    <Display result="0" />
-    <ButtonPanel />
-  </div>
-);
+class App extends Component {
```

Comment:
**[OPTIONAL]** 
per the convention of container/component pattern, components with their own state ought to be in the container folder, whilst other stateless ones which depends on props in the component folder